src/bitmapfont_wqy.h

```diff
@@ -1,6 +1,10 @@
-#include "bitmapfont.h"
+#ifndef EP_BITMAPFONT_WQY_H
```

Comment:
seems like this one is not updated.
(missing `GENERATED FILE` stuff)

Reply:
For some reason, this file is not in the ruby script so it's not a generated file??

Reply:
I found the python script, updated  and reran it.